src/components/Scrollbar.tsx

```diff
@@ -45,12 +61,13 @@ export default function ({
     const numberOfViews = node.scrollHeight / node.clientHeight
     setThumbHeight(100 / numberOfViews)
   }, [])
+
```

Comment:
I guess it could be removed

Reply:
Doesn't the code above already listens for resizes?

Reply:
It listens for changes inside the scroll